docs/user-guide/system-admin/configuration/infinispan.md

```diff
@@ -1,6 +1,9 @@
 # Infinispan for caching
 
-Zanata uses Infinispan to manage some of its internal data caches. Configuration for these caches happens in JBoss' `standalone/configuration/standalone.xml`.
+Zanata uses Infinispan to manage its internal data caches and search indexes. Configuration for these caches happens in JBoss' `standalone/configuration/standalone.xml`. There are two different caches that need to be configured for Zanata:
+
+1. Hibernate search Indexes
```

Comment:
This should probably say Hibernate caches. I think Hibernate Search indexes are a different kettle of fish.